tests/langchain/test_langchain_model_export.py

```diff
@@ -3152,3 +3140,33 @@ def test_load_chain_with_model_config_overrides_saved_config(chain_model_signatu
             "context: {context}\nQuestion: {question}",
             "response": "Databricks",
         }
+
+
+@pytest.mark.skipif(
+    Version(langchain.__version__) < Version("0.0.311"), reason="feature not existing"
+)
+@pytest.mark.parametrize("streamable", [True, False, None])
+def test_langchain_model_streamable_param_in_log_model(streamable, fake_chat_model):
+    for model_type in lc_runnables_types():
+        with mock.patch("mlflow.langchain._save_model"), mlflow.start_run():
+            model = mock.MagicMock(spec=model_type)
```

Comment:
can we use real models?

Reply:
Why?

Reply:
Just because it works fine with mocks doesn't mean it does with real models, right?

Reply:
We shouldn't use mocks to cut corners or write less code.